presentation/src/main/java/com/kdjj/presentation/view/adapter/RecipeEditorListAdapter.kt

```diff
@@ -3,51 +3,58 @@ package com.kdjj.presentation.view.adapter
 import android.view.LayoutInflater
 import android.view.ViewGroup
 import android.widget.ArrayAdapter
+import androidx.lifecycle.findViewTreeLifecycleOwner
 import androidx.recyclerview.widget.ListAdapter
 import androidx.recyclerview.widget.RecyclerView
 import com.kdjj.presentation.R
 import com.kdjj.presentation.databinding.ItemEditorAddStepBinding
 import com.kdjj.presentation.databinding.ItemEditorRecipeMetaBinding
 import com.kdjj.presentation.databinding.ItemEditorRecipeStepBinding
 import com.kdjj.presentation.model.RecipeEditorItem
+import com.kdjj.presentation.viewmodel.recipeeditor.RecipeEditorViewModel
 
-class RecipeEditorListAdapter :
+class RecipeEditorListAdapter(private val viewModel: RecipeEditorViewModel) :
     ListAdapter<RecipeEditorItem, RecyclerView.ViewHolder>(RecipeEditorItemCallback()) {
 
-    class RecipeMetaViewHolder(private val binding: ItemEditorRecipeMetaBinding) :
+    inner class RecipeMetaViewHolder(private val binding: ItemEditorRecipeMetaBinding) :
         RecyclerView.ViewHolder(binding.root) {
 
-        fun bind(item: RecipeEditorItem.RecipeMeta) {
-            val adapter = ArrayAdapter(
-                binding.root.context,
-                R.layout.item_editor_recipe_type,
-                arrayOf("한식", "양식", "중식", "일식")
-            )
-            binding.spinnerEditorRecipeType.adapter = adapter
+        init {
+            binding.imageViewEditorRecipe.clipToOutline = true
+        }
+
+        fun bind(item: RecipeEditorItem.RecipeMetaModel) {
+            binding.viewModel = item
+            binding.editorViewModel = viewModel
```

Comment:
구체적인 결합 클래스를 알 수 없을때는 값을 넣고 `executePendingBindings()`를 해야 바로 반영이 됩니다!
refer : https://developer.android.com/topic/libraries/data-binding/generated-binding?hl=ko#dynamic_variables
```suggestion
            binding.viewModel = item
            binding.editorViewModel = viewModel
            binding.executePendingBindings()
```

Reply:
`executePendingBindings`는 다음 프레임 표시하기 전에 밀려있는 binding을 모두 처리하도록 하는 함수인것으로 알고 있는데, 이 곳에서 꼭 이렇게 처리되어야할 이유가 있을까요??